### DISCUSSION TITLE
Should we "officially" recommend pipx for non-installer users?

### DISCUSSION THREAD

**Discussion by carlosperate** [mu-editor/mu#1985] [Ideas & New Features]
In the user tracker I've been personally recommending to use pipx to install Mu in cases where the users encountered issues installing it in the system Python installation.

https://pypa.github.io/pipx/

pipx automatically installs Python applications in their own isolated virtual environment, and adds the executable to the PATH so that it can be easily launched like any other app. So it makes it really easy to install Mu in a way that is also easy to launch:
```
$ pipx install mu-editor
```
And then:
```
$ mu-editor
```

Rather than the more advanced way of:
```
$ virtualenv .venv
$ source .venv/bin/activate
$ pip install mu-editor
```
And then every time you want to use it
```
$ cd path_to_venv
$ source .venv/bin/activate
$ mu-editor
```

Does anybody have any reservations to add it to the "Install with Python" instructions as an alternative to using a "normal" virtual environment?
https://codewith.mu/en/howto/1.1/install_with_python

**Comment by tjguk** [comment]
I'm fine with that. (I've never used it but from your description it seems like a good solution)
